package.json

```diff
@@ -29,6 +29,7 @@
     "jsdom": "^9.6.0",
     "less": "^2.2.0",
     "less-loader": "^2.0.0",
+    "lodash.isnil": "^4.0.0",
```

Comment:
Rather than do this piecemeal should we either use:
https://github.com/lodash/babel-plugin-lodash or 
https://www.npmjs.com/package/lodash-webpack-plugin